src/main/java/com/epam/lab/paymentsystem/service/impl/UserServiceImpl.java

```diff
@@ -7,15 +7,12 @@
 import com.epam.lab.paymentsystem.entities.enums.Roles;
 import com.epam.lab.paymentsystem.exception.LoginAlreadyExistsException;
 import com.epam.lab.paymentsystem.service.UserService;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class UserServiceImpl implements UserService {
 
-    @Autowired
```

Comment:
Why did you remove @Autowired?

Reply:
UserDAO, RoleDAO are use in constratort UserServiceImpl(UserDAO userDAO, UserDAO userDAO) that's why is not necessary to mark it @Autowired.